### ISSUE TITLE
Builder is generating Nodes without Labels

### ISSUE THREAD

**Issue by Franccescalee** [neo4j-labs/llm-graph-builder#463]
While exploring our graph, I noticed that there are some nodes that are Chunk nodes (they have a text property and an embedding) but they are missing the "Chunk" label. I'm not sure if this is a bug or not but I thought Labels in Neo4j were required?

**Comment by jexp** [comment]
That should not happen /cc @praveshkumar1988 
Do you by chance have a file to reproduce?

**Comment by praveshkumar1988** [comment]
@jexp No, data in DB as well file to reproduce. @Franccescalee Please provide the file and DB creds if possible, to replicate.
Thank you

**Comment by Franccescalee** [comment]
@praveshkumar1988 We loaded quite a few documents so I am unsure which one it was from. I will load a new DB with the same documents and see if I can reproduce.

**Comment by Franccescalee** [comment]
@praveshkumar1988 & @jexp I could not reproduce the issue so we must have done it somehow with a bad query. I'm closing this ticket. Thanks!
